config/crd/kustomization.yaml

```diff
@@ -9,6 +9,7 @@ resources:
 - bases/consul.hashicorp.com_serviceintentions.yaml
 - bases/consul.hashicorp.com_ingressgateways.yaml
 - bases/consul.hashicorp.com_terminatinggateways.yaml
+- bases/consul.hashicorp.com_clusters.yaml
```

Comment:
Should this be _meshes.yaml? (And the rest in this file as well?)

Reply:
good catch!!